org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/serializer/SerializerFragment2.xtend

```diff
@@ -456,12 +460,28 @@ class SerializerFragment2 extends AbstractStubGeneratingFragment {
 				
 				@�Inject�
 				protected void init(�IGrammarAccess� access) {
+					�IF partitions.size > 1�
+						�FOR partition : partitions.indexed�
+							init�partition.key�(access);
+						�ENDFOR�
+					�ELSE�
+						grammarAccess = (�grammar.grammarAccess�) access;
+						�FOR group : allAmbiguousTransitionsBySyntax�
+							match_�group.identifier� = �group.elementAlias.elementAliasToConstructor�;
+						�ENDFOR�
+					�ENDIF�
+				}
+			  
```

Comment:
this space gets always generated (test with `gradlew -PuseJenkinsSnapshots=true generateTestLanguages`)

Reply:
```
diff --git a/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/testlanguages/serializer/ReferenceGrammarTestLanguageSyntacticSequencer.java b/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/testlanguages/serializer/ReferenceGrammarTestLanguageSyntacticSequencer.java
index 57885f254..ed38b0d6b 100644
--- a/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/testlanguages/serializer/ReferenceGrammarTestLanguageSyntacticSequencer.java
+++ b/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/testlanguages/serializer/ReferenceGrammarTestLanguageSyntacticSequencer.java
@@ -23,7 +23,7 @@ public class ReferenceGrammarTestLanguageSyntacticSequencer extends AbstractSynt
        protected void init(IGrammarAccess access) {
                grammarAccess = (ReferenceGrammarTestLanguageGrammarAccess) access;
        }
-
+
        @Override
        protected String getUnassignedRuleCallToken(EObject semanticObject, RuleCall ruleCall, INode node) {
                return "";
```

Reply:
its actually the inconsitent use of tab and space

Reply:
I fixed the indentation